src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -256,7 +262,7 @@ private void saveSchematic(@Nonnull String id) {
                 }
             }
         } catch (IOException e) {
-            getLogger().log(Level.SEVERE, e, () -> "Failed to load file: \"" + id + ".schematic\"");
+            ExoticGarden.getInstance().getLogger().log(Level.SEVERE, e, () -> "Failed to load file: \"" + id + ".schematic\"");
```

Comment:
You're getting the instance of this class??? why not just keep it as `getLogger`